docs/generated/packages/web/executors/file-server.json

```diff
@@ -64,7 +64,8 @@
       "spa": {
         "type": "boolean",
         "description": "Redirect 404 errors to index.html (useful for SPA's)",
-        "default": false
+        "default": false,
+        "x-priority": "important"
```

Comment:
Is this really important? Seems like quite a niche option. Would love to hear you reasoning here.

Reply:
It is not niche. It is quite common to serve static with `spa` options because of 404 errors due to SPA routing. Now, we default this to `false` because most projects have their own redirects infrastructure with their hosting. However, this is useful to surface for the consumers to serve (test) their static files locally.